deploy/catalog_resources/community/7.8/kiecloud-operator.7.8.0.clusterserviceversion.yaml

```diff
@@ -482,6 +492,8 @@ spec:
     name: rhpam-businesscentral-monitoring-rhel8
   - image: registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8@sha256:34ccf7889abce588bf0db5956b1747d785934dc5630135094949318bef49a70e
     name: rhpam-smartrouter-rhel8
+  - image: registry.redhat.io/rhpam-7/rhpam-process-migration-rhel8:7.7.0
```

Comment:
does this image exist in 7.7.0? and will one exist for 7.7.1? If not, we need to only add for 7.8.0 releases or later.

Reply:
Removed 7.7.0 and 7.7.1

Reply:
The image creation request is for 7.8.0